packages/traits/traits.0.1.0/opam

```diff
@@ -0,0 +1,40 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Common traits for generic functionality"
+description: "Common traits for generic functionality"
+maintainer: ["Yurii Rashkovskii"]
+authors: ["Yurii Rashkovskii"]
+license: "ISC"
+homepage: "https://github.com/yrashk/ocaml-traits"
+bug-reports: "https://github.com/yrashk/ocaml-traits/issues"
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "alcotest" {with-test & >= "1.5.0"}
+  "ppx_inline_alcotest" {with-test & >= "1.0.0"}
+  "dune" {build & >= "3.2.0"}
+  "utop" {build & >= "2.9.1"}
+  "ocamlformat" {build & >= "0.21.0"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {pinned}
```

Comment:
```suggestion
  ["dune" "subst"] {dev}
```

Reply:
@mseri thank you for the review; I've processed other comments locally to conform to your recommendations; however, I am unsure how to address this one. The OPAM file is generated by Dune, and I am not sure what might have caused this and how to fix it. Any pointers?

Reply:
This was fixed in dune 2.7. You can change the dune requirements, using dune language 2.7 or greater in the dune-project file, otherwise is ok to fix it manually at release time just for the opam-repository

Reply:
Thank you. WOuld it be appropriate to set dune's lang to 3.2?